plugins/CoreAdminHome/Commands/MigrateAnnotations.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Commands;
+
+use Piwik\Common;
+use Piwik\Container\StaticContainer;
+use Piwik\Option;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugins\SitesManager\Model;
+use Piwik\Updater;
+use Piwik\Updater\Migration\Factory as MigrationFactory;
+
+/**
+ * Command to migrate annotations from option table to a separate db table.
+ */
+class MigrateAnnotations extends ConsoleCommand
+{
+    protected function configure()
+    {
+        $this->setName('core:matomo550-migrate-annotations');
+        $this->addOptionalValueOption('chunk-size', null, 'How many annotations to migrate per SQL insert per site', 20);
+        $this->setDescription(
+            'Only needed for Matomo 5.5.0-b1 upgrade. ' .
```

Comment:
```suggestion
            'Only needed for Matomo 5.5.0-b2 upgrade. ' .
```

The version has changed by now :see_no_evil:

Reply:
good catch